### ISSUE TITLE
Index.html reference not getting updated

### ISSUE THREAD

**Issue by ramonck** [smysnk/gulp-rev-all#64]
My index file did get it's name updated: index.2e6ac32e.html

But the references inside are incorrect.

I see that the references in my css files did get updated correctly.

I see that the references in my js files got updated correctly

I see that the files that are in the revall ignore rule don't get the links/resources updated, I don't want the name changed but I want the references ok :).

OBS: My index.html is not in revall ignore rule.

**Comment by smysnk** [comment]
I hope you are verifying these results on your harddrive and not what you're observing on s3. Gulp-rev-all is not responsible for responsibilities ranging beyond the scope of the plugin.  Please also ensure the destination directory is empty before performing gulp-rev-all step.

**Comment by ramonck** [comment]
How should I do it? Should I do gulp.dest right after revall?

**Comment by smysnk** [comment]
Please provider your gulp configuration.

**Comment by ramonck** [comment]
The Gulp config

``` 
var publisher = awspublish.create(aws);
var headers = {'Cache-Control': 'max-age=0, no-transform, public'};

gulp.task('aws', function () {
  gulp.src('dist/**')
  .pipe(revall({ ignore: [/^\/favicon.ico$/g, /^\/app.manifest/g, /^\/sitemap.xml/g] }))
  .pipe(gulp.dest('./s3'));
}); 
```

**Comment by ramonck** [comment]
I did a seach in .awspublish* file and I didn`t find the files which are referenced within my index.html file, all of which didn`t get updated, not sure if rev-all works with this file.

**Comment by ramonck** [comment]
the turn around I did for when gulp-rev-all doesn't map the files I need or in my index.html case I used the gulp-replace and replaced the files with the files I found using gulp-tap.

Doing gulp-rev-all locally I find it the best process before publish, so I can check and verifiy if everything is ok before I publish it.

After that I'm doing publish process.

Best,

**Comment by louisremi** [comment]
Same problem here, I worked around the problem by disabling minifyHTML in my build. Should I open a different bug?

**Comment by hivre** [comment]
I can confirm the "solution" by @louisremi .  disabling gulp-minify-html did the trick.

**Comment by smysnk** [comment]
Should be fixed in v0.8.0, please give it a try.
